pandas-stubs/core/series.pyi

```diff
@@ -216,13 +216,43 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     @overload
     def __new__(
         cls,
-        data: IntervalIndex[IntervalT],
+        data: IntervalIndex[Interval[int]],
         index: Axes | None = ...,
         dtype=...,
         name: Hashable | None = ...,
         copy: bool = ...,
         fastpath: bool = ...,
-    ) -> Series[IntervalT]: ...
+    ) -> Series[Interval[int]]: ...
+    @overload
```

Comment:
would be nice to add the TODO again

Reply:
Same issue - included the changes from the other PR, so I've reverted that here in commit https://github.com/pandas-dev/pandas-stubs/pull/494/commits/fb512037cf8ce9e5ca7ef956e78c62156d0966ba